lib/async_hooks.js

```diff
@@ -352,6 +354,7 @@ module.exports = {
   executionAsyncId,
   triggerAsyncId,
   executionAsyncResource,
+  asyncWrapProviders: ObjectFreeze({ ...asyncWrap.Providers }),
```

Comment:
```suggestion
  asyncWrapProviders: ObjectFreeze({ __proto__: null, ...asyncWrap.Providers }),
```

Reply:
Should maybe include a comment explaining why this is necessary?